scripts/gen-i18n.js

```diff
@@ -89,6 +125,28 @@ function getTranslationsJs(file) {
                 // had to use a _td to compensate) so is expected.
                 if (tKey === null) return;
 
+                // check the format string against the args
+                // We only check _t: _tJsx is much more complex and _td has no args
+                if (node.callee.name === '_t') {
+                    try {
+                        const placeholders = getFormatStrings(tKey);
+                        for (const placeholder of placeholders) {
+                            if (node.arguments.length < 2) {
+                                throw new Error(`Placeholder found ('${placeholder}') but no substitutions given`);
+                            }
+                            const value = getObjectValue(node.arguments[1], placeholder);
+                            if (value === null) {
+                                throw new Error(`No value found for placeholder '${placeholder}'`);
+                            }
+                        }
+                    } catch (e) {
+                        console.log();
```

Comment:
Empty log?

Reply:
Ah, these are actually intentional to log blank lines which makes the output scan a bit better (I went with a separate log call vs putting a literal newline in the log string so the program looks a bit more like the output).

Reply:
Fair enough!